content/en/docs/tasks/traffic-management/egress/egress-tls-origination/index.md

```diff
@@ -194,17 +194,138 @@ are being sent by inspecting [Server Name Indication (SNI)](https://en.wikipedia
 The _SNI_ field is sent unencrypted during the TLS handshake. Using HTTPS prevents the attackers from knowing specific
 topics and articles but does not prevent an attackers from learning that `edition.cnn.com` is accessed.
```

Comment:
prevent an attackers --> prevent attackers
It is not your fault. Please also help to fix it.